catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/ProcessingDetailsImpl.java

```diff
@@ -82,6 +83,20 @@ public ProcessingDetailsImpl(SourceProcessingDetails details, String sourceId) {
     return warnings;
   }
 
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
+    if (o == null || getClass() != o.getClass()) return false;
+    if (!super.equals(o)) return false;
+    ProcessingDetailsImpl that = (ProcessingDetailsImpl) o;
```

Comment:
Rename the variable "that" to increase readability.

Reply:
What do you suggest as a more readable name?

Reply:
Spoke with @kentmorrissey and I am fine with leaving this name as is.